samples/caf_sensor_manager/src/events/config_event.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _CONFIG_EVENT_H_
```

Comment:
This event is not needed. It adds complexity for no reason.

Reply:
As in previous answer.